.github/workflows/run-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
Why?

Reply:
Vitest 1.0 depends on Vite 5, which does not run on Node 16

Reply:
I am not sure if dropping support (essentially) for Node v16 is a good idea. The current version of Vitest works and there's really no need to upgrade.